blockware/lib/VirtualBlox/wasm.cpp

```diff
@@ -1,3 +1,4 @@
+#if defined(EMSCRIPTEN)
```

Comment:
Since this file is inside `lib/VirtualBlox`, platformio will try to build it. This `#if` makes sure it's only built from wasm/emscripten builds